website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,220 @@
+
+##### Page-level sidebar
+
+Page-level sidebars can be used for more complex data sets and if filtering content of the entire page is required and can be oriented on the left side of the viewport.
+
+![Vertical orientation of the filter bar](/assets/patterns/filter-patterns/layout-sidebar-left-page-level.png =559x*)
+
+##### In-line sidebar
+
+In-line sidebars can be used when the number of filterable parameters is minimal and if space permits.
+
+![Vertical orientation of the filter bar on the left](/assets/patterns/filter-patterns/layout-sidebar-left-inline.png =559x*)
+
+### Applied filters
+
+Display **applied filters** and provide the user with a method to clear all filters at once. Applied filters should be represented using a [Tag](/components/tag) component, which allows for the individual dismissal of a filter value.
+
+![Applied filters](/assets/patterns/filter-patterns/applied-filters.png =450x*)
+
+#### Positioning with filter bars
+
+Applied filters should be positioned between the data set and the filter bar with a `16px` gap between elements.
+
+![Applied filters with filter bar positioning](/assets/patterns/filter-patterns/applied-filters-positioning-filter-bar.png =559x*)
+
+In the case of a filter sidebar, the applied filters should be positioned directly above the dataset with a `16px` gap.
+
+![Applied filters with filter sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar.png =559x*)
+
+If the filters are positioned in a page-level sidebar, sometimes displaying the applied filters isn't necessary because they are implied in the sidebar. An action to clear all of the filters should be included within the sidebar itself.
+
+![Applied filters with clear all in the sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar-clear-all.png =559x*)
+
+### Global filter functions
+
+Depending on the fetching method and complexity of the data set, global filter functions can be used to expose methods that effect the entire data set. Global functions should be differentiated from the primary filters and aligned to the end of the data set. Examples include:
+
+- Manually refreshing the data set
+- Triggering a [Flyout](/components/flyout) with more complex filters
+
+#### Multiple global filter functions
+
+![Multiple global functions](/assets/patterns/filter-patterns/global-filter-functions.png =874x*)
+
+#### Single global function
+
+![Single function](/assets/patterns/filter-patterns/global-filter-functions-single.png =874x*)
+
+In the case a multiple filter functions, use the [Segmented Group](/components/segmented-group). If only a single function is necessary, use the secondary [Button](/components/button#secondary).
+
+### Pagination
+
+Use pagination to break down the filtered data set into pages. For more details, refer to the [Pagination](/components/pagination) guidelines.
+
+![Pagination example](/assets/patterns/filter-patterns/pagination-layout.png =559x*)
+
+## Empty state
```

Comment:
The "Empty state" section feels less important than filtering methods and how to display results. I recommend moving it down after the "Displaying selected filters" section instead.

Also, this doesn't need to be addressed in this iteration, but I'd like us to make a note that we should add documentation to help designers create UIs that actively help users avoid an empty state. One way to do that is to show the # of results available for a particular value, which is shown in some of your examples and the videos, but isn't documented.